modules/tensor_mechanics/src/userobjects/NodalPatchRecoveryBase.C

```diff
@@ -174,3 +174,17 @@ NodalPatchRecoveryBase::finalize()
   libMesh::Parallel::pull_parallel_vector_data(
       _communicator, query_ids, gather_data, act_on_data, ex);
 }
+
+bool
+NodalPatchRecoveryBase::isBlockRestrictionSame(
+    const std::set<SubdomainID> & aux_kernel_subdomains) const
+{
+  std::set<SubdomainID> result;
+  std::set_symmetric_difference(blockIDs().begin(),
+                                blockIDs().end(),
+                                aux_kernel_subdomains.begin(),
+                                aux_kernel_subdomains.end(),
+                                std::inserter(result, result.end()));
+
+  return result.size() == 0 ? true : false;
+}
```

Comment:
oops I missed this. Why don't we just use `operator==`?

Reply:
Ended up deleting this method since there is block subset functionality in the parent objects.